src/MainWindow.vala

```diff
@@ -133,39 +137,44 @@ public class Mail.MainWindow : Hdy.ApplicationWindow {
             }
         });
 
-        paned_start = new Gtk.Paned (Gtk.Orientation.HORIZONTAL);
-        paned_start.pack1 (folders_list_view, false, false);
-        paned_start.pack2 (conversation_list, true, false);
+        paned_start = new Gtk.Paned (Gtk.Orientation.HORIZONTAL) {
+            shrink_start_child = false,
+            shrink_end_child = false,
+            wide_handle = true
```

Comment:
Is there a reason why you added the `wide_handle` here?

Reply:
I hoped this would fix some style issues with the separators because the black border on the left side of the conversation list is shorter than on the right side but it didn't and I'm not even sure whether the black borders belongs to the separators or to something else entirely. Any idea about the origin of these borders and how to fix their size?